Octokit/Models/Request/NewAuthorization.cs

```diff
@@ -11,12 +11,35 @@ namespace Octokit
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public class NewAuthorization
     {
+        // TODO: I'd love to not need this
+        public NewAuthorization()
+        {
+        }
+
+        public NewAuthorization(string note, string[] scopes)
+        {
+            Scopes = scopes;
+            Note = note;
+        }
+
+        public NewAuthorization(string note, string[] scopes, string fingerprint)
+        {
+            Scopes = scopes;
```

Comment:
I'm wary of passing in arrays given they can be mutated. At the very least, we should probably set `Scopes` to a copy of the arry, right?